src/usbasp.c

```diff
@@ -399,6 +399,28 @@ static int usbasp_transmit(const PROGRAMMER *pgm,
   return nbytes;
 }
 
+static int check_for_port_argument_match(const char *port, char *bus, char *device, char *serial_num) {
+
+  if (verbose) {
```

Comment:
pmsg_notice() only outputs sth if `verbose` is one or greater. Hence, the `if` test is unnecessary